packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -533,6 +533,41 @@ aar artifact already exists, continuing...''',
     createArtifactProgress.complete();
   }
 
+  /// Zips and uploads a release xcframework to the Shorebird server.
+  Future<void> createIosFrameworkReleaseArtifacts({
```

Comment:
minor and feel free to ignore but since this likely won't be used by other commands might be nice to leave it as part of the command logic.